src/othello_UI/Game.java

```diff
@@ -35,10 +35,16 @@
 	public Date getStart() throws ParseException {
 		return new SimpleDateFormat("yyyy-MM-dd HH:mm:ss").parse(this.StartString);
 	}
+	public Date getEnd() throws ParseException {
```

Comment:
Dates don't work well with the CSV so we have to have them as strings, this gets the string and turns it into a Java Date we can work with.